ccd-definition/ComplexTypes/AllocationDecision.json

```diff
@@ -12,7 +12,7 @@
     "LiveFrom": "01/01/2017",
     "ID": "AllocationDecision",
     "ListElementCode": "allocationDecision_LabelReason",
-    "ElementLabel": "${allocationProposal.proposalReason}",
+    "ElementLabel": "It should be a ${allocationProposal.proposal} as there are previous proceedings which were heard by a ${allocationProposal.proposal}",
```

Comment:
I don't thikn it's correct - the static content you are building here is just prototype random text - it should be taken from reason as previous implementation - please check with BAs.

Reply:
Again, the previous story i done had this in it, so i changed it back as this was the way it was originally in the prototype from BAs. It got changed back to display something wrong for some reason!

Reply:
Discussed and changed